backend/src/models/Project.ts

```diff
@@ -25,10 +26,17 @@ const ProjectSchema = new Schema<IProject>(
     description: {
       type: Schema.Types.String,
       default: null,
+      indexed: true,
     },
     techs: {
       type: [Schema.Types.ObjectId],
       ref: "tech",
+      indexed: true,
+    },
+    starrers: {
+      type: [Schema.Types.ObjectId],
+      ref: "user",
+      indexed: true,
```

Comment:
need `default: []` or else existing project documents will be missing this field in the response

Reply:
Fixed